style.css

```diff
@@ -0,0 +1,104 @@
+/* styles for the skeleton project begins here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Inter, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  background: #e5e5e5;
+}
+
+#skeleton {
+  width: 375px;
+  margin: 0 auto;
+  background: #fff;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 28px;
+  padding-top: 27px;
+}
+
+p#welcome {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+#menu {
+  width: 16px;
+  height: 10px;
+}
+
+#page {
+  background-image: url("/images/skeleton.svg");
```

Comment:
- [ ] As the last reviewer requested Kindly make sure your background image displays properly. There is an error in displaying.